aspnetcore/tutorials/min-web-api.md

```diff
@@ -628,7 +628,7 @@ Verify you can post and get all fields except the secret field.
 
 ### Configure JSON serialization options
```

Comment:
I really don't like all the H3's, I think a bullet list would be better and more maintainable.

Reply:
Bulleted list here makes sense to me.